src/main/java/io/codelex/oop/Cars/CarService.java

```diff
@@ -0,0 +1,154 @@
+package io.codelex.oop.Cars;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class CarService {
+
+    List<Car> cars = new ArrayList<>();
+
+
+    public void addCarToList(Car carName) {
+        cars.add(carName);
+    }
+
+    public void removeCarFromList(Car carName) {
+        cars.remove(carName);
+    }
+
+    public void returnListOfCars() {
+        for (Car allCars : cars) {
+            System.out.println(allCars);
+        }
+    }
+
+    public void returnByEningeType(EngineType type) {
+        for (Car engine : cars) {
+            if (engine.getEngine().equals(type)) {
+                System.out.println(engine.getName() + " have a " + type + " engine");
+
+            }
+        }
+    }
+
+    public void returnOldCars(int yearOfManufacture) {
+        for (Car yearOf : cars) {
+            if (yearOf.getYearOfManafacture() <= yearOfManufacture) {
+                System.out.println(yearOf.getName() + " is the same year or its older than " + yearOfManufacture);
+            }
+        }
+
+    }
+
+    public void returnExpensiveCars(int price) {
+        for (Car amount : cars) {
+            if (amount.getPrice() > price) {
+                System.out.println(amount.getName() + " is more expensive than " + price);
+            }
+        }
+    }
+
+    public void returnCheapestCars(int price) {
+        for (Car amount : cars) {
+            if (amount.getPrice() < price) {
+                System.out.println(amount.getName() + " is cheaper than " + price);
+
+            }
+
+        }
+    }
+
+    public void returnCarsWithThreeManufaturer() {
```

Comment:
arī ka variable varētu padot to 3

Reply:
šeit biju domājis pašai metodei padot, tādā veidā izveidojot viņu generic 
piemēram `returnCarsByManufacturerCount(int manufacturerSize)`
tādā veidā tev nebūs jātaisa jauna metode ja gribēsi pēkšņi piemēŗam ar vismaz 5 manufacturers